src-gui/src/renderer/rpc.ts

```diff
@@ -354,7 +373,7 @@ export async function initializeContext() {
     enable_monero_tor: useMoneroTor,
   };
 
-  logger.info("Initializing context with settings", tauriSettings);
+  logger.info("Initializing context with settings" + tauriSettings);
```

Comment:
add a space here

Reply:
I prefer using the ` templating syntax

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals